redirects.yaml

```diff
@@ -750,6 +750,7 @@ rss\.xmlsubscribe=feed/?: "/blog/feed"
 s/ubuntu.*/?: "/blog"
 screenshots/?: "/desktop"
 screenshots/document_view/?: "/desktop"
+security/certifications/?: "/security/compliance-automation"
```

Comment:
I understand that this works for all the existing links that point to /security/certifications. But wouldn't it be better if we just changed those places too? When you want to know where a page is being linked, it's easier to find it if we changed all the links. Otherwise you would have to find the row in redirects first and then search for a non-existent path. 
I would only add a row to redirects just to make sure someone who bookmarked the previous link still has an access to the page.

Reply:
Yes, this is a good suggestion. I will leave in the redirect for bookmarks and external links.